test-jest/chart/ComposedChart.spec.tsx

```diff
@@ -45,60 +46,27 @@ describe('<ComposedChart />', () => {
     expect(container.querySelectorAll('.recharts-bar .recharts-bar-rectangle')).toHaveLength(1);
   });
 
-  // test.skip('Generate point scale for x-axis, when has no <Bar /> in the ComposedChart', () => {
-  //   const wrapper = render(
-  //     <ComposedChart width={800} height={400} data={data} margin={{ top: 20, right: 20, bottom: 20, left: 20 }}>
-  //       <XAxis dataKey="name" />
-  //       <YAxis />
-  //       <CartesianGrid stroke="#f5f5f5" />
-  //       <Area type="monotone" dataKey="amt" fill="#8884d8" stroke="#8884d8" />
-  //       <Line type="monotone" dataKey="uv" stroke="#ff7300" />
-  //     </ComposedChart>,
-  //   );
-
-  //   expect(wrapper.state().xAxisMap[0].realScaleType).to.equal('point');
-  // });
-
-  // test.skip('Generate point scale for x-axis, when has <Bar /> in the ComposedChart', () => {
-  //   const wrapper = mount(
-  //     <ComposedChart width={800} height={400} data={data} margin={{ top: 20, right: 20, bottom: 20, left: 20 }}>
-  //       <XAxis dataKey="name" />
-  //       <YAxis />
-  //       <CartesianGrid stroke="#f5f5f5" />
-  //       <Area type="monotone" dataKey="amt" fill="#8884d8" stroke="#8884d8" />
-  //       <Bar dataKey="pv" barSize={20} fill="#413ea0" />
-  //       <Line type="monotone" dataKey="uv" stroke="#ff7300" />
-  //     </ComposedChart>,
-  //   );
-
-  //   expect(wrapper.state().xAxisMap[0].realScaleType).to.equal('band');
```

Comment:
Cannot read component state directly. With RTL you have to read the DOM, and I couldn't find anything different related to this state.